prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -232,8 +232,18 @@ open class Receiver(
      * Validate the object and return null or an error message
      */
     fun consistencyErrorMessage(metadata: Metadata): String? {
-        // TODO: Temporary workaround for full-ELR as we do not have a way to load schemas yet
-        if (topic == Topic.FULL_ELR) return null
+        if (topic == Topic.FULL_ELR) {
+            if (conditionFilter.isNotEmpty()) {
+                conditionFilter.forEach {
+                    if (it.contains(".exists") || !it.contains(".intersect")) {
+                        return "Condition filter must be an intersect and cannot evaluate the intersect, " +
+                            "for example: conditionFilter: [\"%testPerformedCodes.intersect('123-0'|'600-7')\"]"
+                    }
+                }
+            }
+
+            return null
```

Comment:
Ouch. This old logic here shorts the rest if the topic is FULL ELR. Write a new bug ticket to fix this.

Reply:
Ticket created here: https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/8568